cmd/lakectl/cmd/cherry_pick.go

```diff
@@ -27,7 +28,7 @@ var cherryPick = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		ref := MustParseRefURI("ref", args[0])
 		branch := MustParseBranchURI("branch", args[1])
-		Fmt("Branch: %s\n", branch.String())
+		fmt.Printf("Branch: %s\n", branch)
```

Comment:
```suggestion
		fmt.Println("Branch: %s", branch)
```

Reply:
kept the format - can switch to use Println and drop the format but it is the same